gdeployfeatures/ctdb/ctdb.py

```diff
@@ -66,6 +67,14 @@ def ctdb_setup(section_dict):
             ctdbnodes = [ctdbnodes]
         section_dict['ctdbnodes'] = "\n".join(ctdbnodes)
         Global.logger.info("Using %s as CTDB nodes" % ctdbnodes)
+    else:
+        # Ensure that the section_dict['nodes'] are IPs and
+        # not hostnames when ctdb_nodes are not specified
+        for s in Global.hosts:
+            if s.isalpha() is True:
+                msg = "Need ips for nodes, found hostnames."
```

Comment:
This message is quite misleading. Say:

Provide ip address in [hosts] section or define ctdb_nodes variable with ip addresses.